### ISSUE TITLE
upgrade to ses 0.11.1 (for "adapt whitelist to XS" etc.)

### ISSUE THREAD

**Issue by dckc** [Agoric/agoric-sdk#2263]
## What is the Problem Being Solved?

ses 0.11.1 was released Jan 21 but `@agoric/install-ses` still depends on 0.11.0 from Nov 3.

In particular, when I run `lockdown()` on XS, I get a couple dozen of these:

```
Removing intrinsics.ArrayBuffer.fromBigInt
Removing intrinsics.ArrayBuffer.fromString
Removing intrinsics.BigInt.bitLength
```

These were suppresed in a Dec 31 change
https://github.com/Agoric/SES-shim/commit/bd0952ac6569985b7a3508b257916e262b837606

and included in the Jan 21 v0.11.1 release
https://github.com/Agoric/SES-shim/commit/392559f8ae8216811dd9bcfe5d64161e0cf42be8

## Description of the Design

In any `package.json` that refers to `ses`, such as `@agoric/install-ses`, change `0.11.0` to `0.11.1`

For bonus points: develop a process for coordinating future upgrades.

## Security Considerations

Hard to tell. I don't see any additional risks in a brief review of the commit logs. @erights said it's "worrisome" that agoric-sdk is showing issues that were addressed in SES-shim "for a long time now".

I don't see any release notes nor CHANGES file in SES-shim. [SECURITY.md](https://github.com/Agoric/SES-shim/blob/master/SECURITY.md) still shows 0.6.0 as the supported version. I suppose I'll raise one or more issues about that separately.

## Test Plan

Run all the usual tests, I suppose.

**Comment by erights** [comment]
Are we still seeing these on XS?

**Comment by dckc** [comment]
no; this was fixed in #2323
